app/views/courses/confirmation.html.erb

```diff
@@ -26,7 +25,9 @@
                 <%= course.level&.humanize %>
               </dd>
               <dd class="govuk-summary-list__actions">
+                <%= link_to new_provider_recruitment_cycle_courses_level_path(@provider.provider_code, course.recruitment_cycle_year, params.to_unsafe_h.merge(goto_confirmation: true)), class: "govuk-link", data: { qa: "course__edit_level_link" } do %>
```

Comment:
This is making my eyes water. Can we use a helper method or something to help make this more readable pls?